### PR TITLE
Add Edit Jam page

### LINKED ISSUE #27: Review Participant submitted statements
User Value:
**As a**: moderator
**I want to:** I want to review all new participant-submitted statements
**So that:** I can check that they're appropriate for the Jam survey and correct any small spelling or grammar errors.

ACs:
- Assuming that moderator can edit statements as much as they want until they've been marked active for voting ("Accept")
- Once they've been accepted for voting, the mod cannot edit any statement any further.

UXD:

Epic: #16

Size:

### PR DESCRIPTION
**Pull request by siame**
I thought I'd submit this as a PR so people can test the preview deployment. I've linked it up to the moderator home page too so you should be able to click through all the way. 
Functionality wise I _think_ everything is here now (apart from adding new statements, but I need to re-use some components for that), let me know if I've missed something.

The few things that I know are missing are styling based:
- Some of the aggregated stats under the Jam description
- Back to overview link positioning
- White background for everything before the Approved/Rejected/New tabs

Closes #27

### CODE REVIEW

File: components/ModeratorNewStatementCard.js [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+      borderColor="gray.200"
+      my={4}
+      backgroundColor="white"
+    >
+      <Text pb={5}>{statement.text}</Text>
+
+      <Flex>
+        <Box>
+          <Text color="gray.600">
+            <ChatIcon></ChatIcon> Participant submitted{' '}
```
Comment (Midnite) [inline components/ModeratorNewStatementCard.js:86]: you can just do `<ChatIcon />` in these cases

File: components/ModeratorNewStatementCard.js [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+            {new Date(
+              statement.createdAt?._seconds * 1000,
+            ).toUTCString()}
+          </Text>
+        </Box>
+
+        <Spacer />
+
+        <Stack direction="row" spacing={2}>
```
Comment (Midnite) [inline components/ModeratorNewStatementCard.js:95]: could do `HStack` instead and then there's no need for direction=row

File: pages/moderator/[jam].js [hunk 3]
```diff
@@ -0,0 +1,266 @@
...
+    setApprovedStatements(
+      jam.statements.filter((statement) => statement.state === 1),
+    );
+    setRejectedStatements(
+      jam.statements.filter((statement) => statement.state === -1),
+    );
+    setNewStatements(
+      jam.statements.filter((statement) => statement.state === 0),
+    );
+  }, [jam, patchSuccess]);
```
Comment (ruiramos) [inline pages/moderator/[jam].js:104]: Are you using this `patchSuccess` as a toggle just to trigger this `useEffect`?

**Review by ruiramos — APPROVED** [review]

File: pages/moderator/[jam].js (hunk 3 above)
Comment (siame) [reply pages/moderator/[jam].js:104]: Yep, I couldn't get it to detect changes on `jam.statements` by including `jam`, or `jam.statements` here, it's a bit of an issue with using an array of objects as an observable: https://stackoverflow.com/questions/63536876/react-hooks-useeffect-not-being-triggered-on-array-of-objects

**Pull request merged**